x-pack/plugins/stack_connectors/public/connector_types/inference/connector.test.tsx

```diff
@@ -183,9 +710,6 @@ const openAiConnector = {
       model_id: 'gpt-4o',
       organization_id: 'test-org',
     },
-    taskTypeConfig: {
-      user: 'elastic',
-    },
```

Comment:
@YulNaumenko To get this jest test passing, I had to remove this. Looks like `user` used to be a task setting for this config but we're not setting it anymore. Is that ok?

Reply:
yes, it's correct. Thank you for cleaning up the code.